gooddata-sdk/gooddata_sdk/catalog/workspace/content_service.py

```diff
@@ -612,6 +628,23 @@ def get_label_elements(
         if filter_by is not None:
             request.filter_by = filter_by.to_api()
 
+        if pattern_filter is not None:
+            request.pattern_filter = pattern_filter
+
+        if complement_filter:
+            request.complement_filter = complement_filter
+
+        if sort_order is not None:
+            request.sort_order = sort_order
+
+        paging_params = {}
+        if offset is not None:
+            paging_params["offset"] = offset
+        if limit is not None:
+            paging_params["limit"] = limit
+
         # TODO - fix return type of Paging.next in Backend + add support for this API to SDK
```

Comment:
Is this comment still valid?

Reply:
Unfortunately it is. Not sure how to fix that at all...